ccd-definition/CaseEventToFields/DefendantResponseLRspec.json

```diff
@@ -539,7 +540,7 @@
     "PageFieldDisplayOrder": 1,
     "DisplayContext": "READONLY",
     "PageID": "RequestedCourtLocationLRspec",
-    "PageShowCondition": "respondent1ClaimResponseTypeForSpec=\"FULL_DEFENCE\" AND responseClaimTrack= \"SMALL_CLAIM\"",
+    "PageShowCondition": "respondent1ClaimResponseTypeForSpec=\"FULL_DEFENCE\"",
```

Comment:
Do we not want to show the court location page in the fast track as well?

Reply:
rectified